src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -12,7 +12,7 @@ user_token: (IDENT | QUOTED_IDENT | string_literal);
 
 name: ( id | tokens_available_for_names | INTEGER_LITERAL);
 id: ( IDENT | QUOTED_IDENT );
-literal: (float_literal | integer_literal | string_literal);
+literal: (float_literal | integer_literal | string_literal | NULL);
```

Comment:
it doesn't.  it allows all literals (including TRUE/FALSE, which are not represented here), but not `NOW()` or `CURRENT_TIMESTAMP`